test/e2e/atlas/decryption_azure_test.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build e2e || (atlas && decrypt)
+//go:build e2e || (atlas && decryptdisabled)
+
+// re-enable test after Azure key is regenerated and wiki updated
```

Comment:
why not `t.Skip("CLOUDP-<>")` in the test?

Reply:
changed, thanks